src/main/kotlin/io/reactivex/rxkotlin/disposable.kt

```diff
@@ -1,12 +1,12 @@
 package io.reactivex.rxkotlin
 
-import io.reactivex.disposables.CompositeDisposable
 import io.reactivex.disposables.Disposable
+import io.reactivex.internal.disposables.DisposableContainer
```

Comment:
This is an internal interface and should not be depended upon outside RxJava.

Reply:
Yes, you're right. Please reject the PR.